### ISSUE TITLE
Training run-time benchmark information/statistics from ECMWF

### ISSUE THREAD

**Issue by artificiallama** [ecmwf/anemoi-core#535]
ECMWF has comprehensive experience in training models using Anemoi. The training time depends on several factors such as the AI model used (number of trainable parameters), number of variables used in the training, number of epochs/steps, learning rate, etc. ECMWF might have a database with all the registered runs. Could ECMWF make the recorded runtimes public along with values of all the factors ?

This information would be useful to Anemoi users.
